docs/openapi/publicKeys-schemas/public-keys.yaml

```diff
@@ -0,0 +1,95 @@
+info:
```

Comment:
are we planning to change the response that comes to us from the microservice? if no, we don't need to redefine all the models.
Check which models will change and redefine only them